src/Angular.js

```diff
@@ -1532,7 +1532,13 @@ function getNgAttribute(element, ngAttr) {
 
 function allowAutoBootstrap(document) {
   var script = document.currentScript;
-  var src = script && script.getAttribute('src');
+
+  // If the `currentScript` property has been clobbered just return false, since this indicates a probable attack
+  if (!(script instanceof window.HTMLScriptElement || script instanceof window.SVGScriptElement)) {
```

Comment:
This fails on IE, which does not support `document.currentScript`. This should probably be:

```js
if (!script) {
  return true;
} else if (!(script instanceof window.HTMLScriptElement || script instanceof window.SVGScriptElement)) {
  return false;
}

var src = ...
```